modules/mono/glue/GodotSharp/GodotSharp/Core/StringExtensions.cs

```diff
@@ -1236,9 +1244,9 @@ public static int NocasecmpTo(this string instance, string to)
         /// <returns>The string padded with zeroes.</returns>
         public static string PadDecimals(this string instance, int digits)
         {
-            int c = instance.Find(".");
+            int c = instance.IndexOf('.');
 
-            if (c == -1)
+            if (c < 0)
```

Comment:
The C++ implementation seems to use [`== -1`](https://github.com/godotengine/godot/blob/9dd6c4dbac70b28e8156255c3a2b78722772b036/core/string/ustring.cpp#L4645) here.